forte/orbital-helpers/ao_helper.cc

```diff
@@ -105,7 +106,7 @@ void AtomicOrbitalHelper::Compute_AO_Screen(std::shared_ptr<psi::BasisSet>& prim
 void AtomicOrbitalHelper::Estimate_TransAO_Screen(std::shared_ptr<psi::BasisSet>& primary,
                                                   std::shared_ptr<psi::BasisSet>& auxiliary) {
     Compute_Psuedo_Density();
-    MemDFJK jk(primary, auxiliary);
+    MemDFJK jk(primary, auxiliary, psi::Process::environment.options);
```

Comment:
Why is this needed?

Reply:
This is due to a recent Psi4 PR (#2848) where an option object is required in the initializer. I think the psi4 solution is not ideal and breaks "the C++ side of psi4 should be a library" principle. So this is just a patch. It turns out we don't seem to use AtomicOrbitalHelper anyway.